dotfiles/install_crictl.sh

```diff
@@ -0,0 +1,17 @@
+if ! [ $(id -u) = 0 ]; then
+    echo "running as non-root and crictl will not be installed"
+    exit 0
+fi
+
+if [ -x "$(command -v crictl)" ]; then
+  rm -rf /etc/crictl.yaml
+  cp /root/dotfiles/crictl.yaml /etc/crictl.yaml
```

Comment:
I had another thought on this. You can't assume that the host's root directory will be used as root directory of the pod (`chroot`):
1. When running the `hacks/ops-pod` script to schedule a privileged pod `chroot` is not run automatically. It just mounts it under `/host`. Only with `-c` command a `chroot` is executed.
2. Also, the privileged pod that the dashboard / `terminal-controller-manager` schedules by default does not run a `chroot` automatically. It just mounts it under `/hostroot`

Reply:
Hi @petersutter , thanks, i've removed usage of root dir and use tee command to write the /etc/crictl.yaml file.
-Neo